packages/common/src/ui/layout/tables/components/Cells/CurrencyCell/CurrencyCell.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Typography } from '@mui/material';
+import { useCurrencyFormat } from '@common/intl';
+import { RecordWithId } from '@common/types';
+import { CellProps } from '../../../columns/types';
+
+export const CurrencyCell = <T extends RecordWithId>({
+  column,
+  rowData,
+  rows,
+}: CellProps<T>) => {
+  const currencyValue = column.accessor({ rowData, rows }) as string;
+  const formattedCurrency = useCurrencyFormat(currencyValue);
```

Comment:
Ah.. I added this `useCurrencyFormat` to be consistent with the other methods.. but it's not! 🙄  What am I doing!